### PR TITLE
seo alt for images on main landing pages

### PR DESCRIPTION
**Pull request by Emmah-lou**
alt images fixed to help with seo

### CODE REVIEW

File: src/components/home/FlagTransition.jsx [hunk 1]
```diff
@@ -24,7 +24,7 @@ const FlagTransition = () => {
           transition={{ duration: 0.75, delay: 0.2 }}
           className="flag-transition__box"
         >
-          <img className="flag-transition__logo-img" src="/assets/svg/Asset3.svg" alt="daa" />
+          <img className="flag-transition__logo-img" src="/assets/svg/Asset3.svg" alt="trans flag a picture of a transgender flag. lgbtq support. lesbian, gay, bisexual, trans & queer ++ " />
```
Comment (BlocksOnChainCode) [inline src/components/home/FlagTransition.jsx:27]: Does the order matter there? 

I assume SEO rules state to go from high volume to low volume?

File: src/components/support/ClothingResources.jsx [hunk 2]
```diff
@@ -15,7 +15,7 @@ const ClothingResources = () => {
         </li>
         <li>
           <div className="logo-wrapper">
-            <img src="https://static.wikia.nocookie.net/gender/images/3/3c/Genderfree_Flag.png" alt="" />
+            <img src="https://static.wikia.nocookie.net/gender/images/3/3c/Genderfree_Flag.png" alt="gender free world icon" />
```
Comment (BlocksOnChainCode) [inline src/components/support/ClothingResources.jsx:18]: Was there any mention of the actual URL being useful for SEO when it comes to this?

**Review by BlocksOnChainCode — APPROVED** [review]
💯 Good! learning the thingys need be learned

File: src/components/support/ClothingResources.jsx (hunk 2 above)
Comment (Emmah-lou) [reply src/components/support/ClothingResources.jsx:18]: not so much in images but back links and also internal links

File: src/components/home/FlagTransition.jsx (hunk 1 above)
Comment (Emmah-lou) [reply src/components/home/FlagTransition.jsx:27]: well you also want to be readable to a screen reader so it needs to make sense

Comment (Emmah-lou) [reply src/components/home/FlagTransition.jsx:27]: trans flag has alot of searches

**Pull request merged**